src/main/resources/db/changelog/changes/01-create-employees-table.yaml

```diff
@@ -0,0 +1,48 @@
+databaseChangeLog:
+  - changeSet:
+      id: 1
+      author: your_name
```

Comment:
nice author)

Reply:
Tell me please if it is normal that I can't use boolean here?

              - column:
                  name: is_deleted
                  type: **TINYINT(1)**
                  defaultValueBoolean: false
                  constraints:
                    nullable: false